migrations/versions/0366_add_sessions_table.py

```diff
@@ -0,0 +1,30 @@
+"""
+
+Revision ID: 6392698ad84a
```

Comment:
Need to update this for consistency with all the other revision files.  Should be the same as line 11.